src/IntervalArithmetic.jl

```diff
@@ -21,6 +21,7 @@ export ×, dot
 import Base:
     +, -, *, /, //, fma,
     <, >, ==, !=, ⊆, ^, <=,
+    isless,
```

Comment:
`isless` is for total order and `<` is for partial order and fall back to `isless`. Therefore we should only implement `<` and come up with a better name for the function implementing the `less` function defined in the standard (`isweaklyless` seems good to me).

Note that we definitely don't want `isweaklyless` equal to `<`, since the former has more or less nothing to do with number comparison.